Configuration/TypoScript/constants.typoscript

```diff
@@ -1,13 +1,4 @@
 plugin.tx_tea {
-  view {
```

Comment:
To make the templates easily overwritable I would suggest to keep the constants section

Reply:
They are already overwritable (if I read [the docs](https://docs.typo3.org/m/typo3/book-extbasefluid/10.4/en-us/b-ExtbaseReference/Index.html#view) correctly): If a TypoScript entry with a higher key is present, it will take precedence.

Reply:
yes they are. However It is considered best practice by most extensions to provide a constant that makes overriding easier then fiddling with keys. Also this gives the advantage that the Constant Editor can be used to set those. The constants come with a description which is also like in-vito documentation. News is doing it like that: https://github.com/georgringer/news/blob/main/Configuration/TypoScript/constants.typoscript#L25, Powermail is doing it like that: https://github.com/einpraegsam/powermail/blob/develop/Configuration/TypoScript/Main/constants.typoscript#L5, SF Event Manager also: https://github.com/derhansen/sf_event_mgt/blob/main/Configuration/TypoScript/constants.typoscript#L14

Also when I am writing an extension extending tea but want to make sure that the sitepackage can still override my extensions templates I would choose a key that lies between 0 (your base template) and 10 (the sitepackage) and therefore do not have to depend on keys that someone writing a sitepackage chose.